sympy/simplify/combsimp.py

```diff
@@ -217,6 +219,232 @@ def explicate(p):
                 denom_gammas.extend(l)
             elif isg is False:
                 denom_others.extend(l)
+        # =========== level >= 2 work: factor absorbtion =========
+        if level >= 2:
```

Comment:
There should not be any gammas in the expression at this point. Which kind of error is arising?